src/Control/Monad/Catch.hs

```diff
@@ -201,6 +199,24 @@ instance MonadThrow STM where
 instance MonadCatch STM where
   catch = STM.catchSTM
 
+instance e ~ SomeException => MonadThrow (Either e) where
+  throwM = Left . toException
+-- |
```

Comment:
Did you mean to put documentation of some sort here? If not, perhaps it's best to condense it down to:

```haskell
-- | @since 0.8.3
```